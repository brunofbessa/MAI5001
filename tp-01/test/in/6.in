I
1
produto1
1.83

L

F
