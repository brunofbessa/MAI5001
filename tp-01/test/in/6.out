Ok
1 - produto1 - 1.83
Fim
